shell/browser/api/electron_api_base_window.cc

```diff
@@ -98,10 +98,15 @@ BaseWindow::BaseWindow(v8::Isolate* isolate,
   window_->AddObserver(this);
 
 #if defined(TOOLKIT_VIEWS)
+  v8::TryCatch try_catch(isolate);
+
   // Sets the window icon.
   gin::Handle<NativeImage> icon;
   if (options.Get(options::kIcon, &icon) && !icon.IsEmpty())
     SetIcon(icon);
+
+  if (try_catch.HasCaught())
+    LOG(ERROR) << "Failed to convert NativeImage";
```

Comment:
Can we print the path here maybe?

Reply:
unfortunately no - we only have access to the path in the converter